### DISCUSSION TITLE
Inquiry About Dynamically Assigned Gradle Version in build.properties.ejs

### DISCUSSION THREAD

**Discussion by venkatesh-i2i** [jhipster/generator-jhipster-quarkus#340] [Q&A] [answered]
I'm aiming to enhance the Gradle version within our project, but I've noticed that the Gradle version in the build.properties.ejs file is dynamically assigned. I'm curious about where this version is being fetched from. Could someone please enlighten me on this matter? Any insights would be greatly appreciated!

**Comment by mraible** [comment, marked as answer]
The values are likely coming from the main generator-jhipster, in https://github.com/jhipster/generator-jhipster/blob/main/generators/gradle/constants.js. What are you hoping to change?
